core/server/fluxruntime.go

```diff
@@ -39,8 +40,18 @@ var (
 	ErrFluxNamespaceNotFound = errors.New("could not find flux namespace in cluster")
 	// ErrListingDeployments no deployments found
 	ErrListingDeployments = errors.New("could not list deployments in namespace")
+
+	DefaultFluxNamespace = getDefaultString("WEAVE_GITOPS_FALLBACK_NAMESPACE", "flux-system")
 )
 
+func getDefaultString(envVar string, fallback string) string {
```

Comment:
```suggestion
func lookupEnv(envVar string, fallback string) string {
```
This makes code using this function a little more human-parseable. Alternatively can call it `getEnvOrDefault` or similar.